README.md

```diff
@@ -185,6 +185,34 @@ The server must be accessible using the `$HOME/id_rsa` SSH Key, and **root** is
 
 **Note:** Running Streisand against an existing server can be a destructive action! You will be potentially overwriting configuration files and must be certain that you are affecting the correct machine.
 
+### Noninteractive Deployment (Advanced) ###
+
+Alternative scripts and configuration file examples are provided for
+noninteractive deployment, in which all of the required information is passed
+on the command line or in a configuration file.
+
+Example configuration files are found under `global_vars/noninteractive`. Copy
+and edit the desired parameters, such as providing API tokens and other choices,
+and then run the appropriate script.
+
+To deploy a new Streisand server:
+
+      deploy/streisand-new-cloud-server \
+        --provider digitalocean \
+        --site-config global_vars/noninteractive/digitalocean-site.yml
+
+To run the Streisand provisioning on the local machine:
+
+      deploy/streisand-local \
+        --site-config global_vars/noninteractive/local-site.yml
+
+To rerun the Streisand provisioning an existing server:
```

Comment:
"rerun" should be "run". Streisand doesn't really support re-provisioning a server at this point. The existing server mode assumes a fresh 16.04 install like the cloud provisioning modes provide.